client/provider/rest/user/user_handler.go

```diff
@@ -0,0 +1,114 @@
+package rest
+
+import (
+	"net/http"
+	"strconv"
+
+	controllers "github.com/PavelDonchenko/bookstorejRPC/client/controllers/user"
+	pb "github.com/PavelDonchenko/bookstorejRPC/client/gen/proto"
+	"github.com/gin-gonic/gin"
+)
+
+type UserInput struct {
+	Id       uint32 `json:"id"`
+	Nickname string `json:"nickname"`
+	Email    string `json:"email"`
+	Password string `json:"password"`
+}
+
+type routerHandler struct {
+	c *controllers.BaseHandler
+}
+
+func NewRouterHandler(c *controllers.BaseHandler) *routerHandler {
+	return &routerHandler{
+		c: c,
+	}
+}
+
+func (h routerHandler) GetAll(ctx *gin.Context) {
+	res, err := h.c.GetAll()
+	if err != nil {
+		ctx.JSON(http.StatusBadRequest, gin.H{"error": err.Error()})
+		return
+	}
+
+	ctx.JSON(http.StatusOK, gin.H{"response": res})
+}
+
+func (h routerHandler) GetOne(ctx *gin.Context) {
```

Comment:
please rename it to `GetUserByID` or `GetUser`

Reply:
fixed